flutter_src/lib/model/entities/action.dart

```diff
@@ -67,7 +67,11 @@ class BatchAction extends SheetAction {
 }
 
 class AppendAction extends SheetAction {
-  static const List<String> VALUE_LIST = ["EMPTY", "TIME", "EMAIL", "CONTENT"];
+  static const List<String> VALUE_LIST = [VALUE_EMPTY, VALUE_TIME, VALUE_EMAIL, VALUE_CONTENT];
+  static const VALUE_EMPTY = "EMPTY";
```

Comment:
wonder how this works when VALUE_* consts are defined after VALUE_LIST

Reply:
Changed the order